packages/remirror__extension-code-block/src/code-block-utils.ts

```diff
@@ -260,82 +261,108 @@ export function codeBlockToDOM(node: ProsemirrorNode, extra: ApplySchemaAttribut
   return ['pre', attributes, ['code', { [LANGUAGE_ATTRIBUTE]: language, style }, 0]];
 }
 
-interface FormatCodeBlockFactoryProps
-  extends NodeTypeProps,
-    Required<Pick<CodeBlockOptions, 'formatter' | 'defaultLanguage'>> {}
+type FormatCodeProps = NodeTypeProps &
+  Required<Pick<CodeBlockOptions, 'formatter' | 'defaultLanguage'>> &
+  Partial<PosProps> & { state: EditorState };
```

Comment:
Following on the the arguments vs command function props mentioned above,

Instead of `& { state: EditorState }` we should use `& CommandFunctionProps`